go.mod

```diff
@@ -24,59 +24,60 @@ require (
 	golang.org/x/time v0.8.0
 	google.golang.org/protobuf v1.35.2
 	gopkg.in/yaml.v3 v3.0.1
-	istio.io/api v1.23.2
-	istio.io/client-go v1.23.2
-	istio.io/istio v0.0.0-20240917190016-da0b3f29e9f9
-	k8s.io/api v0.30.3
-	k8s.io/apiextensions-apiserver v0.30.3
-	k8s.io/apimachinery v0.30.3
-	k8s.io/client-go v0.30.3
-	k8s.io/kubectl v0.30.3
-	k8s.io/utils v0.0.0-20240502163921-fe8a2dddb1d0
-	sigs.k8s.io/controller-runtime v0.18.3 // Since we are using the Istio library directly, we should also use the controller-runtime version that is used by Istio.
+	istio.io/api v1.24.0-rc.0.0.20241101200753-9397ebf09c3a
+	istio.io/client-go v1.24.0
+	istio.io/istio v0.0.0-20241105211601-8825a6b7f8c9
+	k8s.io/api v0.31.1
+	k8s.io/apiextensions-apiserver v0.31.1
+	k8s.io/apimachinery v0.31.1
+	k8s.io/client-go v0.31.1
+	k8s.io/kubectl v0.31.1
+	k8s.io/utils v0.0.0-20240921022957-49e7df575cb6
+	sigs.k8s.io/controller-runtime v0.19.3 // Since we are using the Istio library directly, we should also use the controller-runtime version that is used by Istio.
```

Comment:
Is there an officially tagged version available `1.24.1`? this one seems to be a release-candidate for `1.24.0`.

Reply:
This is what Istio uses in the 1.24.1 tag. That's usually how it it 😢

Reply:
https://github.com/istio/istio/blob/5c178358f9c61c50d3d6149a0b05a609a0d7defd/go.mod#L97